flutter/news/lib/features/frontpage/presentation/bloc/articles_cubit.dart

```diff
@@ -30,7 +30,7 @@ class ArticlesCubit extends Cubit<ArticlesState> {
           ),
         )
       },
-      failure: (failure) => emit(const ArticlesState.error()),
+      failure: (failure) => emit(ArticlesState.error(error: failure.toString())),
```

Comment:
What could we do here so that we can get the message by doing `failure.message`? Perhaps `InternalFailure` should contain a `message` field? It could make sense because every class implementing it at the moment has it.

For example:
```

abstract class InternalFailure {
  final String message;

  InternalFailure(this.message);
}
```

Reply:
updated